crates/rpc/rpc-eth-api/src/helpers/state.rs

```diff
@@ -325,7 +325,7 @@ pub trait LoadState:
             if block_id == Some(BlockId::pending()) {
                 // for pending tag we need to find the highest nonce in the pool
```

Comment:
we need to update the docs here to reflect that

Reply:
Thanks for the review! Fixed the comment.